interview-questions.md

```diff
@@ -8,62 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: A parameter is declared when creating a function as a input and a argument is used when calling that function as a input 
 
-  Researched answer:
+  Researched answer: a parameter is a variable that is created and passed into a function, they allow you to use arguments when the function is ran
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer: the method .map takes three parameters: value, index and one more
 
-  Researched answer:
+  Researched answer: .map takes three parameter value, index, and array. value is the values in the array, index is the location of the value and array is a copy of the original array
 
 
 
 3. What is the difference between map and filter?
 
-  Your answer:
+  Your answer: the map method returns a new array of the same length and filter returns a new array with the elements from the old array that match the specified parameters
 
-  Researched answer:
+  Researched answer: The map method calls a function for every array element allowing the elements to be manipulated and then returns the values in a new array. The filter method returns an array containing only elements from a selection of items from the original array.
 
 
 
 4. What is the difference between a function and a method?
 
-  Your answer:
+  Your answer: a method is a function but it can be predefined in a class and called on only certain data types
 
-  Researched answer:
+  Researched answer: functions are code written to complete a specific task and is defined by the function keyword and enclosed in curly braces. methods are created as part of an object but also has the function properties. 
 
 
 
 5. What is object destructuring?
 
-  Your answer:
+  Your answer: I dont know. 
```

Comment:
This is not a recommended answer in an interview.  In the event that you do not know, try to answer a simpler problem like what is an object and mention that you will do some research and get back to the interviewer.  "I don't know" shuts the door of communication in an interview.